core/admin/admin.php

```diff
@@ -750,6 +751,26 @@ public function admin_action_new_post() {
 		die;
 	}
 
+	public function admin_action_site_settings_redirect() {
+		check_admin_referer( 'elementor_action_site_settings_redirect' );
+
+		if ( ! current_user_can( 'edit_theme_options' ) ) {
+			wp_die( esc_html__( 'You do not have sufficient permissions to access this page.', 'elementor' ) );
+		}
+
+		$active_tab = filter_input( INPUT_GET, 'active-tab', FILTER_SANITIZE_ENCODED ) ?? null;
```

Comment:
@ManorHazaz you can remove the `?? null`, because that is already handled by `filter_input`